apps/toolkit-app/src/pages/index.tsx

```diff
@@ -38,12 +38,12 @@ const UserInfo = () => {
   } else {
     return (
       <>
-        <Link href={Routes.SignupPage()} passHref>
+        <Link href={Routes.SignupPage()} passHref legacyBehavior>
           <a className="button small">
             <strong>Sign Up</strong>
           </a>
         </Link>
```

Comment:
A [few lines below](https://github.com/blitz-js/blitz/pull/4031/files#diff-0c8c3ff8ba9db952cedca80e2cf4f0783409959d0ec0ff4b4c2ce3e99f409204L93-L96) you remove the wrapped anchor-tag for the `/projects` link. I take that as a hint that maybe the backwards compatibility is not needed anymore. If so, lets get rid of all those wrapped anchor tags since this is one of the best changes in Next 13 :-).

```suggestion
        <Link href={Routes.SignupPage()} passHref className="button small">
          <strong>Sign Up</strong>
        </Link>
```

Reply:
Yup this could be done in a different PR